<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuwasd">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionManual_refresh"/>
    <addaction name="actionManual_TLE_update"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionRadio"/>
    <addaction name="actionRotator"/>
   </widget>
   <addaction name="menuwasd"/>
   <addaction name="menuConnect"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTLE_Sources">
   <property name="text">
    <string>TLE Sources</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionManual_refresh">
   <property name="text">
    <string>Manual refresh</string>
   </property>
  </action>
  <action name="actionManual_TLE_update">
   <property name="text">
    <string>Manual TLE update</string>
   </property>
  </action>
  <action name="actionRadio">
   <property name="text">
    <string>Radio</string>
   </property>
  </action>
  <action name="actionRotator">
   <property name="text">
    <string>Rotator</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
